server/lib/src/value.rs

```diff
@@ -380,7 +383,7 @@ pub enum PartialValue {
     UiHint(UiHint),
     Passkey(Uuid),
     DeviceKey(Uuid),
-    // The label, if any.
+    ECDHKey, // The label, if any.
```

Comment:
I don't think it needs a partialvalue? If there are no query selectors rn, then we don't need this.